test/wald_wolfowitz.jl

```diff
@@ -0,0 +1,37 @@
+using HypothesisTests, Test
+using Distributions
+
+@testset "Wald-Wolfowitz test" begin
+
+@testset "Independent Observations" begin
+    # Get set of independent observations
+    x = randn(1000)
+    tst = WaldWolfowitzTest(x)
+
+    # Should not be significant dependence
+    @test pvalue(tst) > 0.05
```

Comment:
This is too broad. Better hardcode the exact result for a given input vector and check it with another implementation. Then you don't need separate tests for independent and dependent observations. However it can make sense to test different inputs, in particular a boolean array and an array with only equal values.

Also test other functions implemented by the test (see what other tests do).